test/api/test-ot-face.c

```diff
@@ -110,12 +110,21 @@ test_ot_face_empty (void)
   test_face (hb_face_get_empty (), 0);
 }
 
+static void
+test_ot_var_axis_on_zero_named_instance ()
+{
+  hb_face_t *face = hb_test_open_font_file ("fonts/Zycon.ttf");
```

Comment:
Zycon was available somewhere else on the repo and as duplicating file costs nothing on git I am copying and using it for the test.